styles.css

```diff
@@ -34,14 +34,62 @@ main {
   max-width: 88%;
 }
 
+.ham {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #000;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+}
+
+.hamburger.active .ham:nth-child(2) {
+  opacity: 0;
+}
+
+.hamburger.active .ham:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.hamburger.active .ham:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
 #logo {
   text-decoration: none;
   color: black;
   font-weight: 600;
 }
 
 .menu-3 {
-  display: none;
+  position: fixed;
+  top: 0;
+  left: -100%;
+  gap: 0;
+  flex-direction: column;
+  text-align: start;
+  transition: 1s;
+  width: 100%;
+  justify-content: center;
+  margin-right: 200px;
+  z-index: 200;
+  background-color: #fff;
+  height: 100%;
+}
```

Comment:
I cannot see the `X` close button.

<img width="186" alt="Screen Shot 2022-06-20 at 9 51 27" src="https://user-images.githubusercontent.com/82426682/174628539-e64f3a9e-76a2-47e0-888f-aa360400bfc0.png">

Please add it.

Additionally, your design doesn't follow the template design.

<img width="159" alt="Screen Shot 2022-06-20 at 9 54 42" src="https://user-images.githubusercontent.com/82426682/174629221-f090ba61-f37f-4868-97e3-889e624915f3.png">

Please fix it.

<img width="841" alt="Screen Shot 2022-06-20 at 9 56 22" src="https://user-images.githubusercontent.com/82426682/174629533-79bd17da-f58d-40e3-998c-ef74bf5e5216.png">